### DISCUSSION TITLE
Troubleshooting Remote Server Connectivity while uploading data to PRIDE Archive

### DISCUSSION THREAD

**Discussion by ypriverol** [PRIDE-Archive/pride-resources#34] [Q&A]
**Troubleshooting Remote Server Connectivity** - If you are having issues connecting to a PRIDE server, there are several steps you can take to troubleshoot the problem. 

**Step 1**: Verify that your network connection is active. Check your network connection and make sure it is active. 

**Step 2**: Check the server’s configuration to make sure that it is set up correctly. This includes verifying that all the necessary ports are open and that the server’s remote access is enabled. 

Below are the two transfer servers for PRIDE FTP and ASPERA -

```bash
FTP: ftp-pride-private.ebi.ac.uk
FTP PORT: 21

```

ASPERA Server - 

```bash
ASPERA: hx-fasp-1.ebi.ac.uk
ASPERA PORT: 33001
```

To access FTP, use the following command in the command line:

```bash 
telnet ftp-pride-private.ebi.ac.uk 21
```

and 
**output** should be –

```
telnet ftp-pride-private.ebi.ac.uk 21
Trying 193.62.193.142...
Connected to ftp-private.ebi.ac.uk.
Escape character is '^]'.
220-Welcome to ftp-private.ebi.ac.uk
220
```

To access ASPERA, use the following command in the command line:
 
```
telnet hx-fasp-1.ebi.ac.uk 33001

```

and 
**output** should be – 

```
Trying 193.62.193.136...
Connected to hx-fasp-1.ebi.ac.uk

```

in case it is not connected or if any other response is output please contact your system administrator.

**Note**: Telnet is a software to check remote server’s accessibility. It is enabled by default in most of the OS. If it is not enabled in your computer  please search how to enable it.

**Comment by rolivella** [comment]
FYI, in our lab for using Aspera protocol, we had to open 33001 TCP & UDP ports and also open TCP for SSH protocol.

**Reply by selvaebi** [reply]
Thanks, @rolivella, In the latest tool, we are specifically using only port 33001 for both UDP transfer and TCP ssh auth. And also we are checking for the connectivity automatically in the tool before login.
